pytorch/buildspec-graviton-1-10.yml

```diff
@@ -0,0 +1,39 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+region: &REGION <set-$REGION-in-environment>
+framework: &FRAMEWORK pytorch
+version: &VERSION 1.10.0
```

Comment:
I'm not sure why you are maintaining per version yaml files. If this is the pytorch version for the current image, then it should be 1.12.0

Reply:
This is for the legacy purpose. In case there's an update for graviton 1.10 release, this is the buildspec yml file we can copy over and use.